docs/zh/latest/plugins/hmac-auth.md

```diff
@@ -155,7 +155,7 @@ x-custom-a:test
 "
 ```
 
-注意：最后一个请求头也需要 + `\n`。
+注意：最后一个请求头也需要 + `\n`。如果 signing_string 中的某一项不存在，也需要使用一个空字符串代替。
```

Comment:
Can you append the note to https://github.com/apache/apisix/blob/a1f2a425d4fcdd99b97197039d967b196ec87178/docs/zh/latest/plugins/hmac-auth.md#L98?